src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -186,7 +186,7 @@ private Guid LaunchAgentProcess(TestPackage package)
             if (debugAgent)
                 agentArgs += " --debug-agent";
             if (workDirectory != string.Empty)
-                agentArgs += " --work=" + workDirectory;
+                agentArgs += " --work \"" + workDirectory + "\"";
```

Comment:
Will this work? If I look at the agent then it looks like it expects a `=` immediately after `--work`.

https://github.com/nunit/nunit-console/blob/a64809c808b134693936c28df201c65a5bb230e2/src/NUnitEngine/nunit-agent/Program.cs#L78-L81

Reply:
Good catch!